public/app/plugins/panel/flamegraph/components/FlameGraph/FlameGraph.tsx

```diff
@@ -99,18 +99,23 @@ const FlameGraph = ({
       ctx.font = 12 * window.devicePixelRatio + 'px monospace';
       ctx.strokeStyle = 'white';
 
+      const processor = getDisplayProcessor({
+        field: valueField!,
```

Comment:
nitpick here. We should probably just check the `valueField` on line 66 and throw if it's not defined with some reasonable error message. It could happen in case the dataFrame is malformed, and better to fail quickly with a reasonable message then later.